TEST_APPS/testcases/nfc/README.md

```diff
@@ -0,0 +1,277 @@
+﻿# NFC tests.
+Internal document.
+
+A CI test suite for NFC component. These tests validate card mbed emulation cases. The key use case is a NFC smart poster for comissioning (mbed scout).
+The SUT (system under test) is the NFC target. Tests exercise the framework and NDEF transactions when a NFC controller driver is used, or when the stack is configured for an NFC EEPROM chip in the system integration.
+
+This project is called CreamScone, which is an ice tea framework based cli-driven python test.
+
+<!-- TOC -->
+- [NFC tests . CONFIDENTIAL](#nfc-tests-confidential)
+- [Overview](#overview)
+- [NFC System Testing high level design](#nfc-system-testing-high-level-design)
+- [Low level design](#low-level-design)
+- [User Guide](#user-guide)
+  - [Test cases](#test-cases)
+  - [cli commands](#cli-commands)
+  - [How to](#how-to)
+<!-- TOC -->
+
+
+# Overview
+A set of tests run in CI, which can provide:
+- Internal confidence
+- Faster itterations
+- More efficient work
+- Clear escallation path
+
+A [Ice-tea](https://github.com/ARMmbed/mbed-os-5-docs/blob/development/docs/tools/testing/testing_icetea.md) based test suite. In it's simplest form, the suite merely drives API's for the NFC tag reader/writer, and validates a tag simulation running on an idle target, allows test cases in discovery, connection and read/write NDEF records.
+
+In order to mitigate the costs associated with inter-op testing, have a component model and use existing frameworks or parts. The [nfcpy](https://nfcpy.readthedocs.io/) Python library is used as the core of the *CreamScone* component which uses a PN53* device [SCL3711 usb reader](https://www.identiv.com/products/smart-card-readers/rfid-nfc-contactless/scl3711/) over USB to read the mbed simulated tag.  Interactions raise events in the mbed application. Connection and read/write events which get handled in user application get wired up to asyncronously return responses and data values to the ice tea framework. Events and data must correspond to each NFC driver Framework API call and event and includes the data transmitted in/out of the emulated tag.
```

Comment:
Could you rephrase those in the active form? It's difficult to understand as is.